Plugin/Magento/Framework/App/AreaList.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\UpwardConnector\Plugin\Magento\Framework\App;
+
+use Magento\Framework\App\Config\ScopeConfigInterface;
+use Magento\Store\Model\ScopeInterface;
+
+class AreaList
+{
+    /**
+     * @var ScopeConfigInterface
+     */
+    protected $scopeConfig;
```

Comment:
This should be private, unless you intend to expose it as part of the classes API.

Reply:
Fixed by d8bde4b